client/my-sites/stats/stats-purchase/styles.scss

```diff
@@ -545,27 +545,15 @@ $button-padding: 8px 32px;
 		line-height: 40px;
 		margin-bottom: 24px;
 	}
-
-	.components-button + button.components-button {
-		margin-left: 12px;
-	}
 }
 
 .stats-purchase-wizard__actions {
-	display: flex;
-	flex-wrap: wrap;
-	gap: 12px;
 
-	margin: 12px 0;
-
-	.components-button + button.components-button {
-		margin: 0;
-	}
-
-	button {
-		@media (max-width: $break-small) {
+	.components-button {
```

Comment:
Nesting multiple class names creates issues when people need to overwrite styling in the future. This makes a selector stronger unnecessarily.

You are not applying any styling rules on lines 550 and 552, so a single class would be a better choice.

Reply:
@grzegorz-cp — I was actually having some trouble to get this working as the existing classes where overriding it. I'll give it another look now and shoot you a ping if I need help getting this right.

Reply:
The styles I'm trying to override are part of the `.components-button` scope with a specificity of `0-1-0`. I've update my selector to `stats-purchase-wizard__actions.components-button` to get a specificty of `0-2-0`. This matches what I'm seeing in the CSS for the `stats-purchase-single__personal-checklist-button`.

Is this more inline with what you were thinking @grzegorz-cp?